apps/etterlatte-grunnlag/src/main/kotlin/grunnlag/GrunnlagHenter.kt

```diff
@@ -72,7 +75,13 @@ class GrunnlagHenter(
                     personopplysning(person, personDTO, GJENLEVENDE_FORELDER_PDL_V1, PersonRolle.GJENLEVENDE)
                 }
 
-            val vergesAdresseInfo = vergesAdresse.await()
+            val relevantVerge = hentRelevantVerge(soekerPersonInfo.person.vergemaalEllerFremtidsfullmakt)
+            val vergesAdresseInfo =
+                if (relevantVerge != null) {
```

Comment:
Foreslår
`
val vergesAdresseInfo = relevantVerge?.let { hentVergesAdresse(persongalleri.soeker, it) }`